### PR TITLE
14635 Added restoration filing codes

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* bcgov/entity#14635

*Description of changes:*
- added restoration filing codes
- updated Storybook

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: src/enums/filing-codes.ts [hunk 1]
```diff
@@ -17,5 +17,9 @@ export enum FilingCodes {
   INCORPORATION_ULC = 'BCINC',
   REGISTRATION_SP = 'FRREG',
   REGISTRATION_GP = 'FRREG',
+  RESTORATION_FULL = 'RESTF', // for BC/BEN/CC/ULC
+  RESTORATION_LIMITED = 'RESTL', // for BC/BEN/CC/ULC
+  RESTORATION_LTD_TO_FULL = 'RESXF', // for BC/BEN/CC/ULC
+  RESTORATION_LTD_EXTEND = 'RESXL', // for BC/BEN/CC/ULC
```
Comment (severinbeauvais) [inline src/enums/filing-codes.ts:23]: According to this from Mihai:

image

File: docs/main.6bb055d5.iframe.bundle.js [hunk 2]
```diff
@@ -1 +1 @@
...ndUpdate()}onEffectiveDateTypeChanged(val){this.isImmediate=val===enums.c.IMMEDIATE,this.isFutureEffective=val===enums.c.FUTURE_EFFECTIVE,this.isImmediate&&(this.datePicker="",this.dateText="",this.selectHour=[],this.selectMinute=[],this.selectPeriod=PeriodTypes.AM,this.$refs.datePickerRef.clearDate()),this.emitIsFutureEffective(this.isFutureEffective),this.emitEffectiveDate(null),this.emitValid()}emitIsFutureEffective(val){}emitEffectiveDate(val){}async emitValid(){const validDateText=!this.isFutureEffective||!!this.dateText;await vue_esm.default.nextTick();const isDateValid=this.$refs.datePickerRef.validateForm(),isTimeValid=this.$refs.form.validate();return this.isImmediate||!!this.effectiveDateType&&isDateValid&&isTimeValid&&!!this.selectHour.length&&!!this.selectMinute.length&&!this.isUnderTime&&!this.isOverTime&&validDateText}};Object(tslib_es6.a)([Object(lib.d)({default:!1})],EffectiveDateTimevue_type_script_lang_ts_EffectiveDateTime.prototype,"parseInitial",void 0),Object(tslib_es6.a)([Object(lib.d)()],EffectiveDateTimevue_type_script_lang_ts_EffectiveDateTime.prototype,"currentJsDate",void 0),Object(tslib_es6.a)([Object(lib.d)()],EffectiveDateTimevue_type_script_lang_ts_Ef
```
Comment (ChrisGabel23) [inline docs/main.6bb055d5.iframe.bundle.js:1]: Is this file auto-generated? It's not possible to review what's changed here.

Comment (severinbeauvais) [reply docs/main.6bb055d5.iframe.bundle.js:1]: Yes, this file is auto-generated as a result of step 10 in the workflow readme.

I usually just check that the changes sort-of match the code changes. I would expect a few more files to change if I had updated a component. If there are a ton of changed Storybook files then maybe something went wrong.

**Review by ChrisGabel23 — APPROVED** [review]

**Review by jonathan-longe — APPROVED** [review]
LGTM

**Review by vysakh-menon-aot — APPROVED** [review]

**Pull request merged**